### DISCUSSION TITLE
能否通过 YAML Front Matter 设置指定的页面不显示顶部 navbar ？

### DISCUSSION THREAD

**Discussion by GanZhiXiong** [ppoffice/hexo-theme-icarus#940] [Q&A]
也就是不显示下图的红框部分：

![image](https://user-images.githubusercontent.com/27394631/134638589-3311ef9a-4f4d-4c5e-a6c6-5d0ea53f21c1.png)

因为通过 front  matter 设置 `head: null` 会报错。
设置 `navbar: null` 并不能隐藏红框部分。

**Comment by ppoffice** [comment]
没有这种配置。
